domains/cbbnbbc.is-cool.dev.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "Project Description",
+    "domain": "is-cool.dev",
+    "subdomain": "cbbnbbc",
+
+    "owner": {
+        "repo": "https://github.com/cbbnbbc/register",
+        "email": "cbbnbbc@huayuworld.org"
+    },
+
+    "record": {
+        "CNAME": "cmliu6.pages.dev"
```

Comment:
Just seeing a landing page for Nginx, can you send a preview of the actual site?

Reply:
https://cmliu6.pages.dev/auto